app/views/nic/_base_form.html.erb

```diff
@@ -28,25 +30,47 @@
                  :help_inline => :indicator,
                  :class => 'interface_domain', :'data-url' => domain_selected_hosts_path,
                  :size => "col-md-8", :label_size => "col-md-3" } %>
-  <%= select_f f, :subnet_id, accessible_subnets, :id, :to_label,
-               { :include_blank => accessible_subnets.any? ? true : _("No subnets")},
-               { :disabled => accessible_subnets.empty? ? true : false,
+  <%= select_f f, :subnet_id, accessible_v4_subnets, :id, :to_label,
+               { :include_blank => accessible_v4_subnets.any? ? true : _("No subnets")},
+               { :disabled => accessible_v4_subnets.empty? ? true : false,
+                 :label => _("IPv4 Subnet"),
                  :help_inline => :indicator,
                  :class => 'interface_subnet', :'data-url' => freeip_subnets_path,
                  :size => "col-md-8", :label_size => "col-md-3" } %>
+  <%= select_f f, :subnet6_id, accessible_v6_subnets, :id, :to_label,
+               { :include_blank => accessible_v6_subnets.any? ? true : _("No subnets")},
+               { :disabled => accessible_v6_subnets.empty? ? true : false,
+                 :help_inline => :indicator,
+                 :label => _("IPv6 Subnet"),
+                 :class => 'interface_subnet6', :'data-url' => freeip_subnets_path,
+                 :size => "col-md-8", :label_size => "col-md-3" } %>
 <% end %>
 <%= text_f f, :ip,
               :class => :interface_ip,
+              :label => _("IPv4 address"),
               :autocomplete => 'off',
               :help_block => link_to(_("Suggest new"), '#', :class => 'suggest_new_ip'),
               :help_inline => popover('',
-                  _("An IP address will be auto-suggested if you have a DHCP-enabled Smart Proxy on the subnet selected above.<br/><br/>The IP address can be left blank when:<br/><ul><li>provisioning tokens are enabled</li><li>the domain does not manage DNS</li><li>the subnet does not manage reverse DNS</li><li>and the subnet does not manage DHCP reservations</li></ul>"),
+                  _("An IP address will be auto-suggested if you have enabled IPAM on the IPv4 subnet selected above.<br/><br/>The IP address can be left blank when:<br/><ul><li>provisioning tokens are enabled</li><li>the domain does not manage DNS</li><li>the subnet does not manage reverse DNS</li><li>and the subnet does not manage DHCP reservations</li></ul>"),
                   :title => _("IP address auto-suggest"),
                   :rel => 'popover-modal',
                   :'data-placement' => "top"
                 ).html_safe,
                 :size => "col-md-8", :label_size => "col-md-3" %>
 
+<%= text_f f, :ip6,
```

Comment:
When I try to click on 'suggest new ip' , it returns 404 because I have not set a MAC yet. The error here should be a bit more informative than just 'not found'